apps/web/layouts/PageLayout/AuthLayout.tsx

```diff
@@ -0,0 +1,59 @@
+import Image from 'next/image'
+import Link from 'next/link'
+import type {PropsWithChildren} from 'react'
+
+import {RedirectTo} from '../../components/common-components'
+import {EnhancedActiveLink} from '../../components/EnhancedActiveLink'
+import {LoadingIndicatorOverlay} from '../../components/loading-indicators'
+import {useSession} from '../../contexts/session-context'
+import {AccountMenu} from './AccountMenu'
+
+interface AuthLayoutProps extends PropsWithChildren {}
+
+export function AuthLayout({children}: AuthLayoutProps) {
+  const [session, {loading: isLoadingSession}] = useSession()
+
+  if (isLoadingSession) {
+    return <LoadingIndicatorOverlay />
+  }
+
+  if (!isLoadingSession && !session) {
+    return <RedirectTo url="/auth" />
+  }
+
+  return (
+    <div className="relative mx-auto flex min-h-screen max-w-screen-xl flex-col px-4">
+      <header className="mt-4 flex shrink-0 items-center justify-between overflow-x-hidden py-4">
+        <Link href="/pipelines" className="inline-flex">
```

Comment:
Agreed. Meta comment - I think this type of comment is actually better in the codebase itself rather than as a PR comment, as it will be there when you look at the code in future.